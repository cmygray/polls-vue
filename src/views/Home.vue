<template>
  <v-container tag="section" class="home">
    <h1>Home</h1>
  </v-container>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component
export default class Home extends Vue {}
</script>
