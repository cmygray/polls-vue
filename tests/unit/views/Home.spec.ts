import { createLocalVue, mount } from '@vue/test-utils';

import Home from '@/views/Home.vue';
import '../setup';

const localVue = createLocalVue();

describe('Home.vue', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = mount(Home, {
      localVue,
    });
  });

  it('renders as expected', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
