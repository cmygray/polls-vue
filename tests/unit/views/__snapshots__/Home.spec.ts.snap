// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue renders as expected 1`] = `
<section class="container home">
  <h1>Home</h1>
</section>
`;
